backend/tracim_backend/lib/core/content.py

```diff
@@ -910,11 +912,22 @@ def _get_all_query(
             resultset = resultset.filter(Content.parent_id==parent_id)
         if parent_id == 0 or parent_id is False:
             resultset = resultset.filter(Content.parent_id == None)
+        if label:
+            resultset = resultset.filter(Content.label.ilike('%{}%'.format(label)))  # nopep8
 
+        if order_by_label:
+            resultset = resultset.order_by(Content.label)
         return resultset
 
-    def get_all(self, parent_id: int=None, content_type: str=CONTENT_TYPES.Any_SLUG, workspace: Workspace=None) -> typing.List[Content]:
-        return self._get_all_query(parent_id, content_type, workspace).all()
+    def get_all(
+            self,
+            parent_id: int=None,
+            content_type: str=CONTENT_TYPES.Any_SLUG,
+            workspace: Workspace=None,
+            label: str=None,
+            order_by_label=False
```

Comment:
what about using something like `order_by_property` with a default value to None and:

```
if order_by_property:
  resultset = resultset.order_by(order_by_property) ?
```

The usage would be `get_all(..., order_by_property=Content.label)`?

@buxx @inkhey what do you think?

Reply:
I agree with that

Reply:
Done.